skbeam/core/mask.py

```diff
@@ -79,7 +79,7 @@ def bad_to_nan_gen(images, bad):
             yield im
 
 
-def threshold_mask(images, threshold, mask=None):
```

Comment:
These are also different in that they _apply_ the mask, where as the functions you are adding _create_ masks